dxcint/README.md

```diff
@@ -15,39 +15,49 @@ self testing.
 #### Adding tests
 See [Adding tests](#adding-tests-1) section
 ```commandline
-Usage: dxcint add [OPTIONS] DXC_REPOSITORY_ROOT
+Usage: dxcint add [OPTIONS] DXC_REPOSITORY_ROOT SUITE CATEGORY
+
+  Add tests to the suite
+  Positional Arguments:
+      DXC_REPOSITORY_ROOT: A root directory of a dxCompiler repository. Should contain build.sbt.
+      SUITE: Test suite name. Usually a team-defined group of tests to be run in each CI/CD step. Check README for the
+              CLI command to extract available suites.
+      CATEGORY: Test category name. Usually a team-defined category that reflects a type of the test.
+              Check dxcint/config/{SUITE_FILE}.json, where the keys are category names.
 
 Options:
-  -d, --directory TEXT  Directory containing workflow fixtures to be added
+  -d, --directory TEXT  Directory containing workflow fixtures to be added.
+                        DEFAULT: current WD
   -x, --extension TEXT  Extension of the test workflow script files. Allowed
-                        are 'cwl', 'cwl.json', 'wdl'
-  -s, --suite TEXT      Test suite name. Usually a team-defined group of tests
-                        to be run in each CI/CD step
-  -c, --category TEXT   Test category name. Usually a team-defined category
-                        that reflects a type of the test
+                        are 'cwl', 'cwl.json', 'wdl'. DEFAULT: wdl
 ```
 
 #### Running tests
 See [Running tests](#running-tests-1) section
 ```commandline
 Usage: dxcint integration [OPTIONS] DXC_REPOSITORY_ROOT
 
+  Run integration test or test suire
```

Comment:
s/suire/suite

Reply:
fixed in [554d916](https://github.com/dnanexus/dxCompiler/pull/312/commits/554d916f62d2434d5c1407215255d8993e8997e7)